ui/src/components/media/remote_control.rs

```diff
@@ -198,9 +198,11 @@ pub fn RemoteControls(cx: Scope<Props>) -> Element {
                 appearance: Appearance::Danger,
                 text: cx.props.end_text.clone(),
                 onpress: move |_| {
+                    // TODO: This needs a a signal sent when you terminate a call before the other user joins
```

Comment:
the TODO is unnecessary. Blink will handle that signal internally. all the library user has to do is leave the call